app/client/src/components/createCommunity.js

```diff
@@ -25,21 +25,23 @@ import { useNavigate } from "react-router-dom";
 import { connect } from "react-redux";
 import PropTypes from "prop-types";
 import { createCommunity } from "../redux/ducks/communityDuck";
-import { Button } from "@mui/material";
+import { Autocomplete, Button, Chip, TextField } from "@mui/material";
 
 const CreateCommunity = (props) => {
   const navigate = useNavigate();
   const [title, setTitle] = useState("");
   const [description, setDescription] = useState("");
   const [rules, setRules] = useState("");
   const [createCommunityLoading, setCreateCommunityLoading] = useState(false);
-
+  const [tags, setTags] = useState([]);
   async function registerCommunity() {
     setCreateCommunityLoading(true);
+    const newTags = tags.map((tag) => ({ tag: tag, count: 0 }));
```

Comment:
is this just taking the tags the user input and formatting them into the array of objects? If so maybe name it formattedTags

Reply:
Exactly. Just reformatting it for the format the API is looking for.

Formatted tags is a better name. I'll make that change.